content_scripts/vimium_frontend.coffee

```diff
@@ -580,6 +581,8 @@ window.refreshCompletionKeys = (response) ->
 
     if (response.validFirstKeys)
       validFirstKeys = response.validFirstKeys
+    if (response.commandKeys)
+      commandKeys = response.commandKeys
```

Comment:
I have; if this is mergable, then I wanted to follow it with other commits from #1775's rebase, which would immediately put this to use. I can push a commit to remove this, and revert it in the next PR, if you'd prefer.